documentation/docs/pipelines/abapEnvironment/configuration.md

```diff
@@ -89,7 +104,7 @@ stages:
     cfServiceManifest: 'manifest.yml'
     cfServiceKeyConfig: 'sap_com_0510.json'
   Clone Repositories:
-    repositoryNames: ['/DMO/REPO']
+    repositoryNamesFiles: ['repositories.yml']
```

Comment:
This parameter is called "repositories". https://github.com/SAP/jenkins-library/blob/eef3bcde601732bf5e00c31768bc9339b880a2f2/resources/metadata/abapEnvironmentCloneGitRepo.yaml#L48
Or am I missing something?

Reply:
noNo,No, you're right. Changed it accordingly